lib/scrabble.rb

```diff
@@ -1,18 +1,28 @@
 class Scrabble
+  attr_reader :point_values
+  def initialize
```

Comment:
Would've preferred that you left this as a method. Functionally they operate the same, so not a big deal, but want to make sure that you start thinking of methods as things that return things.

Not an issue that I would worry too much about if you had created this from scratch (6 one way, half dozen the other), but try to get used to using the existing code as is if there's not a good reason to change it.